.github/workflows/interface-unit-tests.yml

```diff
@@ -250,7 +300,7 @@ jobs:
            || fromJSON(needs.setup-ci-load.outputs.matrix-max-parallel).default
          }}
       matrix:
-        group: [1, 2, 3, 4, 5]
+        group: ${{ needs.setup-ci-load.outputs.enable_split == '1' && fromJSON('[1, 2, 3, 4, 5]') || fromJSON('[1]') }}
```

Comment:
I also found out about the necessity of this syntax the hard way in my PR last week 😅

Reply:
I hated this so much....

Reply:
On the bright side, we learn every day 🙄